### ISSUE TITLE
[Enhancement] Provide a way to configure port range in ContainerBuilder

### ISSUE THREAD

**Issue by dipak-pawar** [arquillian/arquillian-cube#749]
Currently we can't provide a port range using annotation having property 
```
@Cube(value = "ftp",
    portBinding = { "37185-90->37185-90/tcp"})
```

I think it's not supported using normal configuration as well. Look at [Portbinding](https://github.com/arquillian/arquillian-cube/blob/master/docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/config/PortBinding.java#L19)

**Comment by lordofthejars** [comment]
No, it is not supported but in Cube it is supported (because Docker Compose supports it) so logic should be there to be reused.

**Comment by dipak-pawar** [comment]
@alex Can you point me to correct class for docker compose as I couldn't find it?

**Comment by lordofthejars** [comment]
Actually a method, could be extracted to class https://github.com/arquillian/arquillian-cube/blob/master/docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/compose/ContainerBuilder.java#L479
